exporter/exporterhelper/internal/obs_report_sender.go

```diff
@@ -21,7 +27,6 @@ import (
 )
 
 const (
-	// spanNameSep is duplicate between receiver and exporter.
```

Comment:
This comment is not useful so I'm not bothered that you've removed it, but it makes reviewing harder when there are unrelated changes mixed in.

(If it were just this one I wouldn't say anything, but there's a bunch of similar changes.)